layouts/black-lives-matter.hbs

```diff
@@ -0,0 +1,227 @@
+    </ul>
+    <p>We will be updating this list periodically.</p>
+  </section>
+
+  <section id="our-support">
+
+  <p>In Node.js, we recognize that there is still an incredible amount of work to be accomplished to address the injustices and inequalities that exist, not only in society as a whole but within the technology industry, and even within our own project. To this point, we have not done enough and recognize that we need to do more. We commit to supporting the Black members of the community by using our collective platforms to amplify your words, or by helping you in your careers, contributing to open source, and seeking other opportunities to help.</p>
+
+ <p>Beginning with our upcoming Collaborator Summit event, and extending beyond, we will be offering mentorship to Black members of the community who wish to become contributors to the Node.js project. If you are interested in this, please send us an email at <a href="mailto:blacklivesmatter@nodejs.org">blacklivesmatter@nodejs.org</a> or direct message us at <a href="https://twitter.com/nodejs">@nodejs</a> on Twitter. We will post more information on how we will be proceeding with this effort as we finalize the details both for the period of the Collaborator summit and beyond that single event.</p>
+    <p>In Node.js, we're going to be working on supporting Black members of the community. If we can help - by using our collective platforms to amplify your words, by helping you in your career or in contributing to open-source, or in any other way - we are here for you.</p>
+
+    <section id="we-are-here-for-you">
+      <ul>
+        <li>Tierney Cyren (<a href='https://twitter.com/bitandbang/status/1267893948149911556?s=20'>message of support</a>, <a href='https://twitter.com/bitandbang'>twitter</a>)</li>
+        <li>Joe Sepi (<a href='https://twitter.com/joe_sepi/status/1268249157317332993'>message of support</a>, <a href='https://twitter.com/joe_sepi'>twitter</a>)</li>
+        <li>James M Snell (<a href='https://twitter.com/jasnell/status/1268300682596986880'>message of support</a>, <a href='https://twitter.com/jasnell'>twitter</a>)</li>
+        <li>Myles Borins (<a href='https://twitter.com/MylesBorins/status/1268317788453683200'>message of support</a>, <a href='https://twitter.com/MylesBorins'>twitter</a>)</li>
+      </ul>
+    </section>
+
+
+    <section id="emails">Please feel welcome to reach out to the <a href="https://github.com/nodejs/node#tsc-technical-steering-committee">Node.js TSC members</a> and <a href="https://github.com/nodejs/community-committee#community-committee-members">Node.js Community Committee members</a> via our email addresses listed in the links, respectively.</section>
+  </section>
```

Comment:
```suggestion
    <section id="we-are-here-for-you">
      <p>Please feel welcome to reach out to the <a href="https://github.com/nodejs/node#tsc-technical-steering-committee">Node.js TSC members</a> and <a href="https://github.com/nodejs/community-committee#community-committee-members">Node.js Community Committee members</a> via our email addresses listed in the links, respectively, or via Twitter DMs to the folks linked below:
      <ul>
        <li><a href='https://twitter.com/bitandbang'>Tierney Cyren</a></li>
        <li><a href='https://twitter.com/joe_sepi'>Joe Sepi</a></li>
        <li><a href='https://twitter.com/jasnell'>James M Snell</a></li>
        <li><a href='https://twitter.com/MylesBorins'>Myles Borins</a></li>
      </ul>
    </section>
  </section>
```

Reply:
update to reflect the (proposed) changes in phrasing, which removes the messages of support and streamlines the content.